exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,12 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+if (danielsRole ==="mentor"){
+  console.log(`Hi, I'm Daniel, I'm a ${danielsRole}`);
+} else {
+  console.log(`Hi, I'm Daniel, I'm a ${danielRole}`);
```

Comment:
I like what you did here! 👍 
If Daniel is anything other than a mentor you'll get the correct role output in the console.